frontend/src/modules/displays/MultiStepWizard.tsx

```diff
@@ -25,6 +30,26 @@ import { commitRequest, commitDraftRequest } from '../../store/controller/action
 import store from '../../store';
 import { DASHBOARD_ROUTE } from '../navigation/constants';
 
+/**
+ * @typedef Data
+ *
+ * Interface for Data
+ *
+ * @prop {string} stateKey Unique identifier for the value
+ * @prop {string} label Label for the value
+ * @prop {string} units Unit measurement of the value
+ * @prop {boolean} isAlarmEnabled Config to set if value is Alarm type
+ * @prop {boolean} isSetvalEnabled Config to set if value is Set type
+ * @prop {number | null} committedSetting Current value
```

Comment:
What is the relationship between `committedSetting` and `setValue` and `setValueActual`?

Reply:
`committedSetting` - Inital Set value when popup is opened (Mostly it is same as `setValueActual` & I think maybe this can be removed)
 `setValue` - Set value which user changes in multi step
`setValueActual` - Current real value from Redux store